Makefile

```diff
@@ -100,8 +109,8 @@ lint: fmt vet
 	git diff --exit-code
 
 # Generate code
-generate: controller-gen
-	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+generate: vet
+	go run sigs.k8s.io/controller-tools/cmd/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
```

Comment:
I feel like we can use the `go generate` approach here, too. What do you think?

Reply:
I have unified all generated stuff into one generate.go file. It takes several seconds now, thus I removed the generate target dependency from Makefile's run and test targets, so it'll start faster without generating. otherwise it's possible run just `make generate run` with the same result

Reply:
Now it's creating the CRDs in the correct folder 👍